README.md

```diff
@@ -1,133 +1,96 @@
 
-The software for the interactive landscape has been extracted to https://github.com/cncf/landscapeapp where it is used for other landscapes as well. This repo includes all of the data and images specific to the CNCF landscapes.
-
-## Trail Map
-
-The Cloud Native Trail Map provides an overview for enterprises starting their cloud native journey.
-
-[![CNCF Trail Map](https://github.com/cncf/trailmap/blob/master/CNCF_TrailMap_latest.png)](https://raw.githubusercontent.com/cncf/trailmap/master/CNCF_TrailMap_latest.png)
+The software for the interactive landscape has been extracted to https://github.com/NREL/mobility_landscapeapp/tree/master/src. This repo includes all of the data and images specific to the mobility data landscape.
 
 ## Interactive Version
 
-Please see [landscape.cncf.io](https://landscape.cncf.io).
+Please see [nrel.github.io](https://nrel.github.io/mobility_landscape/).
 
 ## New Entries
 
-* [Cloud native](https://github.com/cncf/toc/blob/master/DEFINITION.md) projects with at least 300 GitHub stars that clearly fit in an existing category are generally included. Put the project in the single category where it best fits.
-* We generally will only list a company's product in one box, to represent its major or best-known offering. We occasionally make exceptions for large companies. Note that if we allowed listing the same product or project in multiple boxes, the over 600 logos on the landscape would multiply to many times that many.
+These are the steps to go through to upload a new entry:
+
+1. After opening a pull request to landscape.yml, add a new entry within this file. Follow the order and format of previous entries.
+1. Please search through the existing Categories and Subcategories before considering to add a new one. Generally speaking, a new subcategory is not created unless mulitple examples could exist within that subcategory.
+1. Survey the filter list to understand the different options available for each variable. Please ensure to follow the same formatting as other entries in the landscape. If none of the filter options fit your entry, add a new variable option.
+1. Please ensure that you additionally add a logo to your new entry as well, see [Logos](#logos)
+
 * We are unlikely to create a new category for products/projects as we'd rather find the best home with the current options.
 * We are generally not including commercial versions of open source software. The exception is that we are showing all Certified Kubernetes implementations.
 * Closed source products need to link to a clear description of your product; no stealth mode companies.
-* Your project or company needs a logo and the logo needs to include the name.
-* Crunchbase organization should be the company or organization that controls the software. That is normally the owner of the trademark, whether or not a trademark has been formally filed.
+* Your project or company needs a logo.
```

Comment:
If there are multiple entries, make sure that the logo includes the name of the entry?